.gitignore

```diff
@@ -87,3 +87,8 @@ ENV/
 
 # Rope project settings
 .ropeproject
+
+#.DS_Store
+.DS_Store
+/.DS_Store
+/./.DS_Store
```

Comment:
There should be a smarter way to ignore this.. `*.DS_Store`?

Reply:
ah yes. Thanks.

Reply:
Its a fake number. The number is not real.

Reply:
i have changed the number to '09'